src/middlewares/loginMW.ts

```diff
@@ -0,0 +1,10 @@
+import {testData} from '../services/authService';
+
+
+export const loginUser = () => {
+    return(dispatch, getState) => {
+        if (testData.email === email && testData.password === password) {
+            Object.assign({},state,{ isLoggedIn:true })
```

Comment:
if fun return another func: ```() => () => ...``` no need to use return keyword ...
Why we using here Object.assign?